lua/gk.lua

```diff
@@ -15,8 +15,39 @@ return function (net_conf, numa_table)
 	local ggu_lcore = table.remove(gk_lcores)
 	gatekeeper.gk_assign_lcores(gk_conf, gk_lcores)
 
+	gk_conf.max_num_ipv4_rules = 1024
+	gk_conf.num_ipv4_tbl8s = 256
+	gk_conf.max_num_ipv6_rules = 1024
+	gk_conf.num_ipv6_tbl8s = 65536
+
+	local fib_entries = {
+		{
+			["ip_prefix"] = "187.73.30.0/30",
+			["action"] = gatekeeper.c.GK_FWD_GRANTOR,
+			["grantor"] = "10.0.0.3",
+		},
+
+		{
+			["ip_prefix"] = "187.73.31.0/30",
+			["action"] = gatekeeper.c.GK_FWD_GRANTOR,
+			["grantor"] = "3ffe:2501:200:3::1",
+			["gateway"] = "10.0.0.254",
```

Comment:
> Just to confirm: in the FIB entries, the Grantor server and gateway should have the same IP version.

Yes. This is a side effect of the second lookup, that is, the lookup of the IP address of the grantor server. If the grantor server is IPv4, the lookup is done on the LPM of IPv4, so it must return an IPv4 gateway. The same goes if the grantor server is an IPv6 address.

On the example, there should not be field "gw_prefix". Just drop this field in both entries and they will be fine. Although the second FIB entry is fine without field "gw_prefix", I'd add that field "ip_prefix" is more _likely_ to be something like "128.197.41.100/24"; notice that the mask is 24 in my example, not 32.